src/packages/cli/src/bin.ts

```diff
@@ -1,15 +1,20 @@
 #!/usr/bin/env ts-node
+
+// hides ExperimentalWarning: The fs.promises API is experimental
+process.env.NODE_NO_WARNINGS = '1'
+
 import {
   arg,
   getCLIPathHash,
   getProjectHash,
   getSchema,
   getConfig,
+  tryLoadEnv,
```

Comment:
`tryLoadEnv` is introduced in this fil but no used?

Reply:
It was here before https://github.com/prisma/prisma/blob/pr/loadEnv-to-sdk/src/packages/cli/src/bin.ts#L74 actually